### PR TITLE
Unmaintained main functions should be removed

### LINKED ISSUE #56: [Refactor] main function in submodules should be removed
As an continual of #53, is it possible for you to give me criterias or a list for choosing main function of which files should/shouldn't be removed? @ssantichaivekin 

Thanks

### PR DESCRIPTION
**Pull request by maxoja**
Resolves #56 .

### CODE REVIEW

**Comment by maxoja** [comment]
Should we keep
- `script01_recongraph_visualize_demo.py`
- `script02_gen_newick_trees.py`
- `script09_test_refactor.py`
in `/empress/clumpr`?

File: empress/clumpr/Histogram.py [hunk 1]
```diff
@@ -168,73 +168,3 @@ def __mul__(self, other):
...
-        assert(histB.histogram_dict == {2:1, 3:1, 5:1})
-        assert(new_hist.histogram_dict == {2:1, 3:1, 4:1, 5:4, 6:2, 7:1, 8:2})
-
-    testShift1()
-    testShift2()
-    testShift3()
-    testCombine1()
-    testCombine2()
-    testProduct1()
-    testProduct2()
```
Comment (maxoja) [inline empress/clumpr/Histogram.py:239]: Are you positive about deleting this instead of moving it elsewhere?

Comment (ssantichaivekin) [reply empress/clumpr/Histogram.py:239]: Could you create a unit test file for this in the test folder?

Comment (maxoja) [reply empress/clumpr/Histogram.py:239]: Let me try if I could

Comment (maxoja) [reply empress/clumpr/Histogram.py:239]: I managed and tested with
running `python -m unittest discover "./test/clumpr"`

**Comment by maxoja** [comment]
> Should we keep
> 
> * `script01_recongraph_visualize_demo.py`
> * `script02_gen_newick_trees.py`
> * `script09_test_refactor.py`
>   in `/empress/clumpr`?

Can you someone help checking these?

**Comment by ssantichaivekin** [comment]
Please merge from master. The file changes list looks like it has stuffs from other pull request.

**Comment by ssantichaivekin** [comment]
Should we keep the scripts -- yes for now. We will move it later.

**Comment by maxoja** [comment]
I am not sure how to do so after once it was already ravelled ...

**Pull request merged**